hawc/apps/mgmt/templates/mgmt/user_assignments.html

```diff
@@ -1,10 +1,84 @@
 {% extends 'crumbless.html' %}
 
 {% block content %}
-    <h2>My assigned tasks</h2>
-    <div id='main'></div>
-{% endblock %}
+<h2>My assigned tasks</h2>
 
-{% block extrajs %}
-    {{ config|json_script:"config" }}
-{% endblock extrajs %}
+{% if show_completed %}
+<a class="btn btn-light" style="margin-bottom: 15px;" role="button" href=".?completed=off">Hide completed and abandoned
```

Comment:
Instead of hard coding margin, use bootstrap utility classes , you can use `mb-4` or something like that: https://getbootstrap.com/docs/4.6/utilities/spacing/